enhancements/baremetal/enable-baremetal-on-other-platforms.md

```diff
@@ -0,0 +1,179 @@
+with the Machine API is required.
+
+### User Stories
+
+#### Story 1 - Current IPI baremetal platform use case
+
+No change.
+
+#### Story 2 - ZTP use case
+
+If the platform is one of "None, OpenStack or vSphere". cluster-baremetal-operator
+is enabled and watching for a Provisioning CR. When one is created it will
+deploy BMO as per normal (with the validation restrictions defined above). If none
+is created then it will not deploy BMO.
+
+### Risks and Mitigations
+
+There is concern that *random* customers will use this feature out of context
+and create support burden. This is why we have not suggested enabling CBO on
+all platforms and with full feature set. However it is still a potential issue.
+
+## Design Details
+
+### Open Questions [optional]
+
+Re ZTP requirements:
+1. Are these all the platforms that are required?
+2. Is just enabling spec.provisioningNetwork=Disabled mode sufficient?
+
+### Test Plan
+
+#### Unit Testing
+
+We will add unit tests to confirm that cluster-baremetal-operator:
+* is enabled on the required platforms.
+* will restrict functionality on these platforms to ProvisioningNetork=Disabled.
+
+#### e2e Testing
+
+It is expected that these use cases will be functionally tested by the Assisted Installer
+team as there are other changes necessary to get the overall solution working.
+
+### Graduation Criteria
+
+#### Dev Preview -> Tech Preview
+
+- Ability to utilize the enhancement end to end
+- End user documentation, relative API stability
+- Sufficient test coverage
+- Gather feedback from users rather than just developers
+- Enumerate service level indicators (SLIs), expose SLIs as metrics
+- Write symptoms-based alerts for the component(s)
+
+#### Tech Preview -> GA
+
+- More testing (upgrade, downgrade, scale)
+- Sufficient time for feedback
+- Available by default
+- Backhaul SLI telemetry
+- Document SLOs for the component
+- Conduct load testing
+
+**For non-optional features moving to GA, the graduation criteria must include
+end to end tests.**
+
+#### Removing a deprecated feature
+
+- Announce deprecation and support policy of the existing feature
+- Deprecate the feature
+
+### Upgrade / Downgrade Strategy
+
+cluster-baremetal-operator will upgrade as it currently does, this is only a
+minor change in functionality.
+
+On the platforms (None, OpenStack and vSphere) where the operator was in a disabled
+state, after been upgraded it will move into an enabled state. However in all but ZTP
+use cases nothing will change as there is no Provisioning CR.
+
+### Version Skew Strategy
+
+None required as this is not dependant on other components.
+
+## Implementation History
+
+This PR is the current WIP implementation: https://github.com/openshift/cluster-baremetal-operator/pull/189
+
+## Drawbacks
+
+There is concern that *random* customers will use this feature out of context
+and create support burden.
+
+## Alternatives
```

Comment:
Another alternative is to additionally distribute baremetal-operator as an optional operator with OLM. It's probably not a good idea, but worth enumerating as an alternative. The main downsides are the complexity of releasing and distributing the same project two different ways, and the potential for install-time confusion or conflict over which method should be used to install it.